collect_stata/write_json.py

```diff
@@ -266,11 +285,11 @@ def generate_stat(data, metadata, study):
 def write_json(data, metadata, filename, study=""):
```

Comment:
Please complete the type hints. Also do not set study to "". An empty study should lead to an error.